runtime/lua/vim/lsp/buf.lua

```diff
@@ -448,8 +448,9 @@ function M.document_symbol(opts)
 end
 
 --- @param call_hierarchy_items lsp.CallHierarchyItem[]?
```

Comment:
```suggestion
--- @param call_hierarchy_items lsp.CallHierarchyItem[]
```
We only call this function in one place, and there's already a `nil` check there.